core/store/src/db.rs

```diff
@@ -32,67 +35,70 @@ impl Into<io::Error> for DBError {
 
 #[derive(PartialEq, Debug, Copy, Clone)]
 pub enum DBCol {
-    ColBlockMisc = 0,
-    ColBlock = 1,
-    ColBlockHeader = 2,
-    ColBlockHeight = 3,
-    ColState = 4,
-    ColChunkExtra = 5,
-    ColTransactionResult = 6,
-    ColOutgoingReceipts = 7,
-    ColIncomingReceipts = 8,
-    ColPeers = 9,
-    ColEpochInfo = 10,
-    ColBlockInfo = 11,
-    ColChunks = 12,
-    ColPartialChunks = 13,
+    /// Column to indicate which version of database this is.
```

Comment:
Shalln't database scheme update be somehow supported by migration script?

Reply:
Unfortunately it is not.

Reply:
Going forward we will have migrations for db inside nearcore. This is first setup where we actually store the version of the database in the database.